project/BNFC.scala

```diff
@@ -81,23 +87,28 @@ object BNFC {
   def bnfcFiles(base: File): Seq[File] = (base * "*.cf").get
 
   lazy val bnfcSettings = {
-    inConfig(BNFCConfig)(Defaults.configSettings ++ Seq(
-      javaSource     := (javaSource in Compile).value,
-      scalaSource    := (javaSource in Compile).value,
-      bnfcNamespace  := "coop.rchain.rholang.syntax",
-      bnfcGrammarDir := baseDirectory.value / "src" / "main" / "bnfc",
-      bnfcOutputDir  := (javaSource in Compile).value,
-      bnfcDocDir     := baseDirectory.value / "doc" / "bnfc",
-      clean          := cleanDir(bnfcOutputDir.value / nsToPath(bnfcNamespace.value)),
-      generate       := {
-        val fullCP = (fullClasspath in BNFCConfig).value
-        bnfcFiles(bnfcGrammarDir.value).foreach { (f: File) =>
-          bnfcGenerateSources(fullCP, f, bnfcOutputDir.value, bnfcNamespace.value)
+    inConfig(BNFCConfig)(
+      Defaults.configSettings ++ Seq(
+        // format: off
+        javaSource     := (javaSource in Compile).value,
+        scalaSource    := (javaSource in Compile).value,
+        bnfcNamespace  := "coop.rchain.rholang.syntax",
+        bnfcGrammarDir := baseDirectory.value / "src" / "main" / "bnfc",
+        bnfcOutputDir  := (javaSource in Compile).value,
+        bnfcDocDir     := baseDirectory.value / "doc" / "bnfc",
+        clean          := cleanDir(bnfcOutputDir.value / nsToPath(bnfcNamespace.value)),
+        generate       := {
+          val fullCP = (fullClasspath in BNFCConfig).value
+          bnfcFiles(bnfcGrammarDir.value).foreach { (f: File) =>
+            bnfcGenerateSources(fullCP, f, bnfcOutputDir.value, bnfcNamespace.value)
+          }
+        },
+        cleanDocs      := cleanDir(bnfcDocDir.value),
+        generateDocs   := bnfcFiles(bnfcGrammarDir.value).foreach { (f: File) =>
+          bnfcGenerateLaTeX(f, bnfcDocDir.value)
         }
-      },
-      cleanDocs      := cleanDir(bnfcDocDir.value),
-      generateDocs   := bnfcFiles(bnfcGrammarDir.value).foreach { (f: File) =>
-        bnfcGenerateLaTeX(f, bnfcDocDir.value)
-      }))
+        // format: on
```

Comment:
Format off/on comments added

Reply:
Added those to retain the align. Opened an issue in scalafmt for that.